### DISCUSSION TITLE
What are you doing now?

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#304] [General]
It's been a while since I saw any commits from you @objeck.

**Comment by objeck** [comment]
I had to rebuild my PC, which was costly, and I am job hunting.

**Reply by ghost** [reply]
Are you done rebuilding your computer yet? Btw, good luck.

**Reply by ghost** [reply]
@objeck Is everything OK now?

**Reply by ghost** [reply]
You seem to not check the Discussions page anymore.

**Reply by ghost** [reply]
It seems I'm being ignored.

**Reply by objeck** [reply]
No, I am trying to push out a new version. I cannot implement 18 enhancements. I will choose 2 for the next release. Adding support for Web Assembly or libffi are big ticket items requiring time and research. 

I will move the outstanding issues to talk about. When you make mass changes to tickets, I need help to keep up.

**Comment by objeck** [comment]
Ok, that is what is supposed to be wrapped. The 'module' project builds as a shared library.

**Reply by ghost** [reply]
I'm not sure if we are mentioning the same thing or not. Please search Google for `using C++ class from C`. If you just changed the makefiles to make a DLL instead of a static library then nothing changed, the DLL is still C++ only (and it's specific to the compiler that compiled it, not any more portable than a static library), it's still not possible to use this DLL from C.

**Reply by ghost** [reply]
Btw, we should discuss it on #303 instead of here.

**Reply by ghost** [reply]
It seems we really misunderstood each other. I mean the C language, not C++. You closed #303 with an example for C++.

**Reply by ghost** [reply]
Btw, I intended to close #303 as not planned, too. So I'm totally OK with that. The native language of Objeck is C++. It's natural to use C++. Trying to wrap the C++ code to be able to use it from C could be done, but it will be inefficient. Building the module code as a shared library has no meaning without wrapping it to be able to use it in C.
